lib/index.js

```diff
@@ -157,6 +158,7 @@ function __(StreamCtor) {
             // check to see if we have a readable stream
             if (_.isFunction(xs.on) && _.isFunction(xs.pipe)) {
                 s = new StreamCtor();
+                s.writable = true;
```

Comment:
The very next line has it being written to by the Node stream `xs`, which apparently doesn't work without this.